backend/app/adapter/db/url.go

```diff
@@ -170,6 +171,101 @@ func (u URLSql) composeParamList(numParams int) string {
 	return parameterStr
 }
 
+// FindURLsByUser finds all urls created by a given user
+func (u URLSql) FindURLsByUser(user entity.User) ([]entity.URL, error) {
+	statement := fmt.Sprintf(`
+SELECT %s, %s, %s, %s, %s
+FROM %s
+INNER JOIN %s ON %s.%s = %s.%s
+WHERE "%s"=$1;`,
```

Comment:
JOIN statements prevent our service scale to more database shards. Let's avoid using it when possible.